htdocs/user/card.php

```diff
@@ -805,7 +805,7 @@
 	{
 		$valuetoshow.=($valuetoshow?', ':'').$langs->trans("HTTPBasicPassword");
 	}
-	if (preg_match('/dolibarr/',$dolibarr_main_authentication))
+	if (! preg_match('/ldap|http/',$dolibarr_main_authentication))
```

Comment:
This will not work as expected.
If authentication mode is "ldap,dolibarr"  (both ldap and dolibarr), we must have password in edition because "dolibarr" is present.
A better solution would be to add a hook to allow a module to show something or not on all the blowk "<tr>" of password.

Reply:
@eldy 
ready : https://github.com/Dolibarr/dolibarr/pull/8583

Reply:
Another solution that does not need any change in core is to have your authentication mode called:

"dolibarr_mymethod"  if you want to have the password field visible for edition

and just

"mymethod" if you don't want

Reply:
@eldy 
otherwise hooks will customize the field and its contents